pysat/utils/_core.py

```diff
@@ -702,8 +702,8 @@ def display_available_instruments(inst_loc=None, show_inst_mod=None,
 class NetworkLock(Lock):
     def __init__(self, *args, **kwargs):
         """Lock manager compatible with networked file systems.
-        For a description of acceptable args and kwargs see:
-        :class:`portalocker.utils.Lock`
+        See portalocker.utils.Lock for more details
+        (:class:`portalocker.utils.Lock`)
```

Comment:
Move it down to a Note section after "Returns"

Reply:
Whoops of course